src/features/profile/components/MyProfileGeneral.tsx

```diff
@@ -8,22 +11,22 @@ import Select, { TOption } from './Select/Select'
 export type TProfileGeneral = {
   editMode: boolean
   isEmpty: boolean
+  nativeCurrency?: string
```

Comment:
Could you add a typescript enum for nativeCurrency?

Reply:
Fixed